### PR TITLE
[luci/pass] Support InstanceNorm with 3D input

### PR DESCRIPTION
**Pull request by ragmani**
This commit will support InstanceNorm with 3D input.

ONE-DCO-1.0-Signed-off-by: ragmani <ragmani0216@gmail.com>

### CODE REVIEW

**Comment by ragmani** [comment]
- For #13700 
- Draft #13717

File: compiler/luci/pass/src/FuseInstanceNormPass.cpp [hunk 1]
```diff
@@ -296,6 +336,40 @@ namespace
  *         |
  *         V
  *       [Out]
+ *-------------------------------------------------------------------
+ * Version_6 (For only 3D I/O)
```
Comment (seanshpark) [inline compiler/luci/pass/src/FuseInstanceNormPass.cpp:340]: if the pattern is same as existing one, can you please add that info, like;

```diff
- * Version_6 (For only 3D I/O)
+ * Version_6 (Same as Version_2, For only 3D I/O)
```

Comment (ragmani) [reply compiler/luci/pass/src/FuseInstanceNormPass.cpp:340]:
```diff
- * Version_6 (For only 3D I/O)
+ * Version_6 (Same as Version_5, For only 3D I/O)
```

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Pull request merged**